simple_history/admin.py

```diff
@@ -162,7 +162,7 @@ def history_form_view(self, request, object_id, version_id):
             'is_popup': False,
             'media': mark_safe(self.media + admin_form.media),
             'errors': helpers.AdminErrorList(form, formsets),
-            'app_label': model._meta.app_label,
+            'app_label': original_opts.app_label,
```

Comment:
I guess this makes sense. Even if the admin is for the history... From the perspective of the admin the app should line up with the original model.